lib/LANraragi/Plugin/Download/EHentai.pm

```diff
@@ -109,10 +110,8 @@ sub provide_url {
         return ( error => "Couldn't proceed with an original size download: <pre>$content</pre>" );
     }
 
-    # If the URL doesn't already have autostart, append start=1 to get an URL that automatically triggers the download.
-    unless ( $finalURL =~ /.*autostart=1.*/gi ) {
-        $finalURL .= "?start=1";
-    }
+    # Append start=1 to get an URL that automatically triggers the download.
```

Comment:
It's not really an append anymore since we're just replacing the query parameters directly now, would be good to edit the comment to reflect that.
```suggestion
    # Set URL query parameters to ?start=1 to automatically trigger the download.
```